DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/theSession/metadata.json

```diff
@@ -0,0 +1,140 @@
+{
+    "Slug": "the-session",
+    "SourceFileName": "TheSession.java",
+    "Title": "The Session",
+    "NonInteractive":  true,
+    "DescriptionLines": [
+        "**The Session**, which is derived from the Document Store, is the primary way your client code interacts with your RavenDB databases.",
+        "",
+        "The Session is a unit of work that represents a single business transaction against a particular database,",
+        "managing change tracking for all the entities that it has either loaded or stored."
+    ],
+
+    "Assets": [
+        {
+            "Title": "The Document Session",
+            "Type" : "link",
+            "Url"  : "https://github.com/ravendb/book/blob/v4.0/Ch02/Ch02.md#the-document-session"
+        },
+        {
+            "Title": "What is a Session",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/4.2/java/client-api/session/what-is-a-session-and-how-does-it-work"
+        },
+        {
+            "Title": "Opening a Session",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/4.2/java/client-api/session/opening-a-session"
+        },
+        {
+            "Title": "The SaveChanges Command",
+            "Type" : "link",
+            "Url"  : "https://github.com/ravendb/book/blob/v4.0/Ch02/Ch02.md#savechanges"
+        },
+        {
+            "Title": "Saving Changes",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/4.2/java/client-api/session/saving-changes"
+        },
+        {
+            "Title": "Sample Program",
+            "Type" : "link",
+            "Url"  : "https://github.com/ravendb/ravendb-jvm-tutorials#session-and-unit-of-work-pattern"
+        }
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Open the Session",
+            "Slug" : "step-1",
+            "DescriptionLines": [
+                "* Direct communication with the database is done through the `Session`.<br>",
+                "  Start by obtaining a new Session object from the Document Store.",
```

Comment:
same here

Reply:
Explanation is about the 'Session Concept' - leaving Capitalized